flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -654,13 +655,39 @@ public TableResult executeSql(String statement) {
 		return executeOperation(operations.get(0));
 	}
 
+	@Override
+	public StatementSet createStatementSet() {
+		return new StatementSetImpl(this);
+	}
+
 	@Override
 	public TableResult executeInternal(List<ModifyOperation> operations) {
-		if (operations.size() != 1) {
-			throw new TableException("Only one ModifyOperation is supported now.");
-		}
+		List<Transformation<?>> transformations = translate(operations);
+		String jobName = extractJobName(operations);
+		Pipeline pipeline = execEnv.createPipeline(transformations, tableConfig, jobName);
+		try {
+			JobClient jobClient = execEnv.executeAsync(pipeline);
+			TableSchema.Builder builder = TableSchema.builder();
+			Object[] affectedRowCounts = new Long[operations.size()];
+			for (int i = 0; i < operations.size(); ++i) {
+				// if only one operation, field name is 'affected_rowcount', else is 'affected_rowcount_$i'
+				String fieldName = "affected_rowcount";
```

Comment:
how about using the target table name as the column name?

Reply:
I think "affected_rowcount" is necessary, how about "affected_rowcount_sink_table_name" ?

Reply:
Seems too long to me, I think most users are aware of this bigint is representing the row count inserted into the table

Reply:
in fact, the table qualified name may be long. in addition, we may need to add catalog name and database name to distinguish between tables with the same name.

Reply:
Yes, that will be long. But if we don't show this, after we display some real row counts in the future, it would be impossible for users to understand the meaning.

Reply:
yes, if there are many sinks.
I will try to extract sink identifier names as the shortest form (also make sure them are unique) to reduce the length of field name